express-notes-api/index.js

```diff
@@ -0,0 +1,209 @@
+const express = require("express")
+const app = express()
+
+const fs = require("fs")
+
+const data = require("./data.json")
+let {nextId,notes}=data
+
+
+app.get("/api/notes",(req,res)=>{
+  let notesArr = []
+
+  for (const prop in notes){
+    notesArr.push(notes[prop])
+  }
+
+  res.status(200).json(notesArr)
+})
+
+app.get("/api/notes/:id",(req,res)=>{
+  if (isNaN(req.params.id) === false){
+    let id = parseFloat(req.params.id)
+
+    const idArr=[]
+    for (const prop in notes) {
+      idArr.push(parseInt(prop))
+    }
+
+    if (Number.isInteger(id)){
+      if(Math.sign(id)===1){
```

Comment:
These two lines would be unnecessary if you modified line 21 to also check is `req.params.id` is greater than zero. Less checks = happy code